glass/lensing.py

```diff
@@ -623,44 +629,65 @@ def deflect(
     alpha
         Deflection values. Must be complex-valued or have a leading
         axis of size 2 for the real and imaginary component.
+    xp
+        The array library backend to use for array operations. If this is not
+        specified, the array library will be determined from the other parameters,
+        if possible.
 
     Returns
     -------
         The longitudes and latitudes after deflection.
 
+    Raises
+    ------
+    ValueError
+        If neither an array nor the array backend ``xp`` are provided.
+
     Notes
     -----
-    Deflections on the sphere are :term:`defined <deflection>` as
-    follows:  The complex deflection :math:`\alpha` transports a point
+    Deflections on the sphere are :term:`deflection` as follows:
```

Comment:
I think put it back as it was. It's essentially doing `[defined](deflection)` if it was a markdown link.